src/wp-includes/class-wp-scripts.php

```diff
@@ -863,6 +863,18 @@ private function get_dependents( $handle ) {
 		return $dependents;
 	}
 
+	/**
+	 * Check if the strategy is valid or not.
+	 *
+	 * @param string $strategy The script strategy.
+	 * @return bool True if strategy part of strategies.
+	 */
```

Comment:
```suggestion
	/**
	 * Determines if a scripts defined loading strategy is valid.
	 *
	 * @param string $strategy The defined script strategy.
	 * @return bool True if the strategy is of an allowed type, false otherwise.
	 */
```